### PR TITLE
Python: Fix checkver and autoupdate

### LINKED ISSUE #158: python: checkver fails
The extractor has that error while trying to find the latest version :
python: couldn't match 'Latest Python 3 Release - Python ([\d.]+)' in https://www.python.org/downloads/windows/

It seems that scoop don't seem to be able to decipher the content correctly...

Some examples similar-ish of what scoop will receive :

- url : https://www.python.org/downloads/windows/
   ```bash
   curl -i https://www.python.org/downloads/windows/ --output -
   ```
   The output looks like
   ```bash
   HTTP/1.1 200 OK
   Connection: keep-alive
   Content-Length: 27045
   Content-Type: text/html; charset=utf-8
   [...]
   Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
   
   ▼♥��w�F�/����(��qvH�o*���8vƹ��↔e&{��♥��♦�$↑☻�����~����↨ Q��.��ܝ�☼◄�ꮮwW��i▲͒�u�]'
   ���g�⌂j��◄^z��{�����S�ț-�8>;ZE�7�↨♠#�J+♀��?‼�#����◄����?[���_+�A�/�w�������↨§L�{
   ```

One solution could be to fall back using the github tag

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
Since it's no longer possible to do some regex directly on the content of the website in commandline easily , I propose to revert back to the github repos and use the tag instead. The tags seems to be accurate enough.

With an issue similar with the autoupdate hash, I removed it and enforce that all downloaded files must be autoupdated and not only the python exe.
## Related Tickets & Documents
Fixes #158
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien: :lock: :poop: 
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### File persistence to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### python

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
